src/player.h

```diff
@@ -1152,6 +1155,8 @@ class player : public Character, public JsonSerializer, public JsonDeserializer
         int cached_turn;
         tripoint cached_position;
 
+        int fell_asleep_turn;
```

Comment:
Hmm, I guess it's hypothetical that somehow the player could save in the middle of sleeping through autosave or something, force quit the game, and then reload they could crash it and we should initialize to -1, for at least good practice's sake.

On the other hand it should never be read without already being initialized since it's automatically set every time the player goes to sleep, and is read immediately at the end of said sleep.